hs/PostResponse.hs

```diff
@@ -144,8 +144,10 @@ checkPost =  do
             H.div ! A.id "spec_misc" $ do 
                 H.p ! A.class_ "code" $ H.em $ H.toHtml $ inqStr
                 H.div ! A.class_ "more-info" $ do
-                    H.input ! A.type_ "text" 
-            H.p ! A.class_ "code" $ H.em "No more than 1.0 FCEs from CSC490H, CSC491H, CSC494H, CSC495H, BCB430Y"
+                    H.input ! A.type_ "text"
+            H.h3 "Notes"
+            H.div ! A.id "notes" $ do
+                H.p "- You may take no more than 1.0 FCE of CSC490H1, CSC491H1, CSC494H1, CSC495H1, BCB430Y1" 
```

Comment:
Just copy the statement from the course calendar: No more than more than 1.0 FCE from CSC490H1, CSC491H1, CSC494H1, CSC495H1, BCB430Y1 may be used to fulfill program requirements